### ISSUE TITLE
提供一个调试好的 pip 环境（python==3.9.16）

### ISSUE THREAD

**Issue by ZenBuilds** [renmada/t5-pegasus-pytorch#58]
absl-py==1.4.0
aiohttp==3.8.4
aiosignal==1.3.1
async-timeout==4.0.2
attrs==22.2.0
bert4torch==0.2.7
brotlipy==0.7.0
cachetools==5.3.0
certifi @ file:///croot/certifi_1671487769961/work/certifi
cffi @ file:///croot/cffi_1670423208954/work
chardet==5.1.0
charset-normalizer==3.0.1
click==8.1.3
cryptography @ file:///croot/cryptography_1677533068310/work
et-xmlfile==1.1.0
fairscale==0.4.13
filelock==3.9.0
flit_core @ file:///opt/conda/conda-bld/flit-core_1644941570762/work/source/flit_core
frozenlist==1.3.3
fsspec==2023.1.0
future==0.18.3
google-auth==2.16.1
google-auth-oauthlib==0.4.6
grpcio==1.51.3
huggingface-hub==0.12.1
idna @ file:///croot/idna_1666125576474/work
importlib-metadata==6.0.0
jieba==0.42.1
joblib==1.2.0
Markdown==3.4.1
MarkupSafe==2.1.2
mkl-fft==1.3.1
mkl-random @ file:///tmp/build/80754af9/mkl_random_1626186066731/work
mkl-service==2.4.0
multidict==6.0.4
nltk==3.8.1
numpy==1.24.2
nvidia-cublas-cu11==11.10.3.66
nvidia-cuda-nvrtc-cu11==11.7.99
nvidia-cuda-runtime-cu11==11.7.99
nvidia-cudnn-cu11==8.5.0.96
oauthlib==3.2.2
openpyxl==3.1.1
packaging==23.0
pandas==1.5.3
Pillow==9.4.0
protobuf==4.22.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser @ file:///tmp/build/80754af9/pycparser_1636541352034/work
pyDeprecate==0.3.1
pyOpenSSL @ file:///croot/pyopenssl_1677607685877/work
PySocks @ file:///tmp/build/80754af9/pysocks_1605305812635/work
python-dateutil==2.8.2
pytorch-lightning==1.4.9
pytz==2022.7.1
PyYAML==6.0
regex==2022.10.31
requests==2.28.2
requests-oauthlib==1.3.1
rouge==1.0.1
rsa==4.9
sacremoses==0.0.53
scikit-learn==1.2.1
scipy==1.10.1
six @ file:///tmp/build/80754af9/six_1644875935023/work
sklearn==0.0
tensorboard==2.12.0
tensorboard-data-server==0.7.0
tensorboard-plugin-wit==1.8.1
threadpoolctl==3.1.0
tokenizers==0.10.3
torch==1.13.1
torch4keras==0.0.6
torchaudio==0.12.1
torchmetrics==0.5.0
torchvision==0.13.1
tqdm==4.64.1
transformers==4.12.0
typing_extensions==4.5.0
urllib3 @ file:///croot/urllib3_1673575502006/work
Werkzeug==2.2.3
yarl==1.8.2
zipp==3.15.0

**Comment by lichengyang666** [comment]
请问有没有高版本transformers==4.30以上的环境呢
